src/main/java/pl/patrycja/ox/ui/ConsoleUI.java

```diff
@@ -1,33 +1,49 @@
 package pl.patrycja.ox.ui;
 
+import pl.patrycja.ox.Language;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
 import java.util.Scanner;
 
 class ConsoleUI implements UI {
 
     private Scanner scanner;
-    private InputChecker inputChecker = new InputChecker();
+    private InputChecker inputChecker;
     private String[] args;
+    private ResourceBundle messages;
 
     public ConsoleUI(String[] args) {
         this.args = args;
         scanner = new Scanner(System.in);
+        inputChecker = new InputChecker(this);
```

Comment:
This is circular dependency, think about which should have which.

Reply:
Yes, I will check it :)